R/trackhub.R

```diff
@@ -206,9 +272,13 @@ TrackHub <- function(uri, create = FALSE) {
     th <- new("TrackHub")
     th@uri <- normURI(uri)
     if (create) {
+        genomeContainer <<- GenomeContainer()
```

Comment:
Again, not sure why `genomeContainer` is a global variable, but replacing it will not work, since the namespace is locked.

Reply:
Both `TrackHub` and `TrackHubGenome` objects need consistent state of genomes. As `genomeField()<-`( in `TrackHubGenome` class)  can update all the fields for a genome. 
For example: If we update `genome` field with `genomeField()<-` value of `genome` field needs to be consistent  across `TrackHub` and `TrackHubGenome` objects in the memory representation.
So to maintain consistent state of genomes, I used global object of `GenomeContainer` class.